src/Web.Api/Features/Telegram/GithubProfiles/ProcessGithubProfileTelegramMessageHandler.cs

```diff
@@ -47,13 +50,41 @@ public async Task<string> Handle(
         ProcessTelegramMessageCommand request,
         CancellationToken cancellationToken)
     {
+        // Handle inline queries
```

Comment:
If I click on Telegram inline suggestion, the system send new message to the chant instead of replacing text. Please change the logic to replace text in the inline message to the GitHub stats